### ISSUE TITLE
Crash with Storage Network

### ISSUE THREAD

**Issue by looney1977** [romelo333/notenoughwands#13]
The game crashes when Storage Network and Not Enough Wands are installed. Removing either solves the problem. I don't know if it's of any importance, but I have only Swapping, Building and Moving wands enabled. Thank you. http://pastebin.com/4QS7e4vf

**Comment by McJty** [comment]
That's weird. Did you also report this to Storage Network?

**Comment by MrGraphic** [comment]
Yes he did but both of us, @MrRiegel and I have been unable to reproduce this. I tried different combinations. Everything is working as expected in both dev and normal enviroment on both sides.
https://github.com/MrRiegel/Storage-Network/issues/14 .

**Comment by MrRiegel** [comment]
maybe a problem with multiMC?
btw: the crash constrained me to install Not Enough Wands. I'm bound to say very cool mod ;)

**Comment by looney1977** [comment]
Thank you, guys, for your effort. Both mods are awesome, keep up the good work.
